ga4gh/datamodel/ontologies.py

```diff
@@ -5,25 +5,38 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import fnmatch
-import os
 import ga4gh.protocol as protocol
 
 
-class FileSystemOntology(object):
+class Ontology(object):
```

Comment:
These are not ontologies, they are maps to links to ontologies;
they should not be called Ontology.

Maciek Smuga-Otto <notifications@github.com> writes:
> In ga4gh/datamodel/ontologies.py:
> 
> >  import ga4gh.protocol as protocol
> >
> >
> > -class FileSystemOntology(object):
> > +class Ontology(object):
> 
> Again, Ontology -> OntologyMap or is it OK as is?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub*
>